ReactWindows/ReactNative/Modules/Cursor/CursorModule.cs

```diff
@@ -0,0 +1,42 @@
+﻿using ReactNative.Bridge;
+using ReactNative.Reflection;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+
+namespace ReactNative.Modules.Cursor
+{
+    class CursorModule : NativeModuleBase
+    {
+        public override string Name
+        {
+            get
+            {
+                return "Cursor";
+            }
+        }
+
+        /// <summary>
+        /// Set a mouse cursor
+        /// </summary>
+        /// <param name="cursor">The name of a system cursor (listed here: https://docs.microsoft.com/en-us/uwp/api/windows.ui.core.corecursortype ).</param>
+        [ReactMethod]
+        public void setCursor(string cursor)
+        {
+            var convertedCursor = EnumHelpers.Parse<CoreCursorType>(cursor);
+            DispatcherHelpers.RunOnDispatcher(() =>
+            {
+                Window.Current.CoreWindow.PointerCursor = new CoreCursor(convertedCursor, 0);
+            });
```

Comment:
Could run into a bad situation here. Right now, `DispatcherHelpers` always runs on the CoreDispatcher for the `MainWindow`, but the logic uses `Window.Current`. Not sure if `Window.Current` is what's in focus or what's correlated to the CoreDispatcher (I presume in-focus). There has been some interest in multi-window support lately, so I want to make sure we don't add too much stuff that needs to be unwound to better support multi-window.